src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -91,3 +93,75 @@ def cluster_projects(projects: Sequence[Project]) -> None:
             metrics.incr(
                 "txcluster.cluster_projects.unclustered", amount=unclustered, sample_rate=1.0
             )
+
+
+@instrumented_task(
+    name="sentry.ingest.span_clusterer.tasks.spawn_span_cluster_projects",
+    queue="transactions.name_clusterer",  # XXX(iker): we should use a different queue
+    default_retry_delay=5,
+    max_retries=5,
+    soft_time_limit=PROJECTS_PER_TASK * CLUSTERING_TIMEOUT_PER_PROJECT,
+    time_limit=PROJECTS_PER_TASK * CLUSTERING_TIMEOUT_PER_PROJECT + 2,
```

Comment:
Sorry for the empty suggestion, deleted it now because it made no sense. The point I was trying to make is that `spawn_span_cluster_projects` is a copy of `spawn_clusterers`, not `cluster_projects`, so it should use the same celery parameters as `spawn_clusterers`:

```python
    default_retry_delay=5,  # copied from release monitor
    max_retries=5,  # copied from release monitor
```

It might also help to rename this function to `spawn_clusterers_span_descs`, to be consistent with `cluster_projects_span_descs`.